### PR TITLE
[AGW][Installation]Fix mtr0 bring up + var cleanup + final reboot

### PR DESCRIPTION
**Pull request by tmdzk**
Signed-off-by: Timothée Dzik <timdzik@fb.com>

## Summary

Fix mtr0 bring up + var cleanup + final reboot

## Test Plan

./agw_installation.sh

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

File: lte/gateway/deploy/roles/ovs_build/vars/all.yaml [hunk 1]
```diff
@@ -1,14 +1,2 @@
-# ovs version on github default v2.8.1
-ovs_version: "v2.8.1"
-# ovs version short in order to retreive patches if any
-ovs_version_short: "2.8.1"
-# patches relative location
-gtp_patch_path: "third_party/gtp_ovs"
```
Comment (xjtian) [inline lte/gateway/deploy/roles/ovs_build/vars/all.yaml:6]: Why did all of these get deleted with no change to the actual role? Did I miss a prior change?

File: lte/gateway/deploy/roles/ovs_deploy/tasks/main.yml [hunk 2]
```diff
@@ -83,39 +75,63 @@
...
+    packages:
+      - magma
+
 - name: Start service openvswitch-switch.
   become: yes
   service:
     name: openvswitch-switch
     state: started
+  tags:
+    - skipfirstinstall
```
Comment (xjtian) [inline lte/gateway/deploy/roles/ovs_deploy/tasks/main.yml:92]: It looks like baremetal provisioning is the only place where the ovs_deploy role is used, why keep these tasks around if we're going to skip them?

**Review by xjtian — CHANGES_REQUESTED** [review]
bouncing with questions

File: lte/gateway/deploy/roles/ovs_deploy/tasks/main.yml (hunk 2 above)
Comment (tmdzk) [reply lte/gateway/deploy/roles/ovs_deploy/tasks/main.yml:92]: I had plan to recycle that role but I can just remove it for now.

File: lte/gateway/deploy/roles/ovs_build/vars/all.yaml (hunk 1 above)
Comment (tmdzk) [reply lte/gateway/deploy/roles/ovs_build/vars/all.yaml:6]: Those are unused in the role. It was just a dumb copy paste

**Review by xjtian — APPROVED** [review]

**Pull request merged**